frontend/src/container/FeedItemWithPreview.jsx

```diff
@@ -120,6 +120,7 @@ export class FeedItemWithPreview extends React.Component {
         <FeedItemHeader
           breadcrumbsList={props.breadcrumbsList}
           content={props.content}
```

Comment:
since content equals to content={serialize(activity.content, ...)} from ActivityList.jsx, couldn't you deduce isPublication from content.contentNamespace?
It would avoid passing the props isPublication

Reply:
The problem would be at publications page where we have contents with the namespace 'publication' that we want to show as contents and not publications.

Reply:
ok I see